<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectOpen</class>
 <widget class="QDialog" name="projectOpen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pshBttnBrowse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pshBttnOpen">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneEdtBrowseProject">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pshBttnCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
